### DISCUSSION TITLE
Send http request to hostname with certificate

### DISCUSSION THREAD

**Discussion by andrewid18** [mobile-dev-inc/Maestro#1795] [Q&A]
Hi, 

I'm new with Maestro automation. I need to call http request, but the hostname require me ssl certificate. Or else I got this error
```
org.mozilla.javascript.WrappedException: Wrapped javax.net.ssl.SSLHandshakeException: Received fatal alert: certificate_required
```

Anyone can help me on how to do it ? Thanks !

**Comment by Sangeeta29071986** [comment]
Try in localhost
